BBBadgeBarButtonItem/BBBadgeBarButtonItem.m

```diff
@@ -142,6 +141,7 @@ - (void)setBadgeValue:(NSString *)badgeValue
         self.badge.backgroundColor      = self.badgeBGColor;
         self.badge.font                 = self.badgeFont;
         self.badge.textAlignment        = NSTextAlignmentCenter;
+        self.badge.layer.masksToBounds  = YES;
 
```

Comment:
@dmga28  Only @dkalachov can answer that. Meanwhile, I manually added the files to my project without Cocoapods to avoid overriding the changes until the pod gets updated.